### ISSUE TITLE
Support android

### ISSUE THREAD

**Issue by abdelkrimkr** [LadybirdBrowser/ladybird#2774]
### Summary

Make android app

### Operating system

Android

### Steps to reproduce

1.
2.
3.
..

### Expected behavior

..

### Actual behavior

..

### URL for a reduced test case

..

### HTML/SVG/etc. source for a reduced test case

```html
..
```

### Log output and (if possible) backtrace

```shell
...
```

### Contribute a patch?

- [ ] I’ll contribute a patch for this myself.

**Comment by diegoiast** [comment]
1.  There is documentation for building on Android did you test it? https://github.com/LadybirdBrowser/ladybird/blob/master/Documentation/BuildInstructionsLadybird.md#android
2. If you are not able to contribute, this issue will be closed IMHO.

**Comment by abdelkrimkr** [comment]
> 1. There is documentation for building on Android did you test it? https://github.com/LadybirdBrowser/ladybird/blob/master/Documentation/BuildInstructionsLadybird.md#android
> 2. If you are not able to contribute, this issue will be closed IMHO.

Can you add apk  to releases ?

**Comment by shlyakpavel** [comment]
Ok, let's put it this way:
> There is documentation for building on Android. Did you test it?

There's no need to test it - it won't work. At the very least, we need to integrate Skia and replace the removed LibArchive to make it functional.

> If you are not able to contribute, this issue will be closed, IMHO.

A tracking issue could be helpful to outline what remains to be done and guide newcomers on the current status of the Android build. I'm not arguing to must be _this_ issue though.

> Can you add an APK to releases?

They cannot, because:
1. We do not provide releases as we're still in the early pre-alpha stage.
2. The Ladybird master branch is currently not buildable for Android.

**Comment by InvalidUsernameException** [comment]
> It would be more helpful if it was more clearly stated that android isn't currently buildable :/

Fair point. Do you want to open a small PR to add a note at the top of [Documentation/BuildInstructionsLadybird.md#android](https://github.com/LadybirdBrowser/ladybird/blob/master/Documentation/BuildInstructionsLadybird.md#android) and [Documentation/EditorConfiguration/AndroidStudioConfiguration.md](https://github.com/LadybirdBrowser/ladybird/blob/master/Documentation/EditorConfiguration/AndroidStudioConfiguration.md)?

**Comment by dotnetCarpenter** [comment]
This issue can probably be closed now that #5053 landed.

**Comment by ADKaster** [comment]
Eh. I'd rather the Android port be in a bit more usable state than it currently is before declaring this issue closed. For example, you can't scroll the viewport, there's no tabs, scaling is dubious, and there's a lot of missing features in the UI process.

**Comment by FaFre** [comment]
I tried to give it a shot, but the android build still seem to have some issues and is not building on the current master:

```bash
ld.lld: error: undefined symbol: rx::DisplayAndroid::DisplayAndroid(egl::DisplayState const&)
>>> referenced by Display.cpp:500 (/home/fafre/development/experimental/ladybird/Build/vcpkg/buildtrees/angle/src/bd4ca4d539-27842c5e34.clean/src/libANGLE/Display.cpp:500)
>>>               Display.cpp.o:(egl::(anonymous namespace)::CreateDisplayFromAttribs(long, long, long, egl::DisplayState const&)) in archive vcpkg_installed/arm64-android/debug/lib/ANGLE.a
```

**Comment by aloeppert** [comment]
> I tried to give it a shot, but the android build still seem to have some issues and is not building on the current master:

Just to echo, I had the exact linking error - I synced master yesterday.

**Comment by Stewarpt** [comment]
Why did the "The android build is broken" banner get removed if the android build is still broken?

**Comment by trflynn89** [comment]
> Why did the "The android build is broken" banner get removed if the android build is still broken?

Because at the time it was removed, the build was working. See https://github.com/LadybirdBrowser/ladybird/pull/5053.

**Comment by Lampe2020** [comment]
I cannot sync the gradle project in Android Studio, it fails to execute `Meta/ladybird.py` with `EACCESS`.   
I have marked the file as executable, and Gemini in the IDE seemed to only mostly randomly prod around and not find anything useful.   
Currently I am trying to see if I myself try to replace `gradle` (i.e. acting in the terminal as if _I_ was `gradle`), if I can get it to build.   
[EDIT]: I couldn't get it to work, sadly.
